api/api.go

```diff
@@ -40,16 +40,18 @@ func init() {
 // Server wraps a qri p2p node, providing traditional access via http
 // Create one with New, start it up with Serve
 type Server struct {
+	lib.Instance
```

Comment:
With the understanding that this refactoring is large and only partially present in this PR, I think it would be good to get rid of `cfg` and `qriNode` fields on this struct, and just access them directly on the instance.